ui/src/pages/Login/styles.module.css

```diff
@@ -0,0 +1,56 @@
+.container {
+  height: 100vh;
+  width: 100vw;
+  background-color: #e2e2e2;
+  background-image: url('../../images/background.svg');
+  background-size: cover;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+.card {
+  background-color: var(--primary-bg);
+  height: 60vh;
+  width: 22vw;
+  border-radius: 40px;
+  filter: drop-shadow(10px 10px 30px rgba(0, 0, 0, 0.25));
+  display: flex;
+  flex-direction: column;
+  justify-content: flex-start;
+  align-items: center;
+  padding: 10vh 0 0 0;
```

Comment:
Can be converted to `filter-top`?

Reply:
Can you explain this one? What do you mean by `filter-top`?

Reply:
My bad, I meant `padding-top`.